local_db/api.py

```diff
@@ -62,15 +62,28 @@ def get_current_request(self, workspace: str, user: User, create=False):
 
     def get_requests_for_user(self, user: User):
         requests = []
-        filter_args = {}
 
-        # if not output checker, can only see your own requests
+        for metadata in RequestMetadata.objects.filter(author=user.username).order_by(
+            "status"
+        ):
+            # to create a request, user *must* have explicit workspace
+            # permissions - being an output checker is not enough
+            if metadata.workspace in user.workspaces:
+                requests.append(self._request(metadata))
+
+        return requests
+
+    def get_outstanding_requests(self, user: User):
```

Comment:
Maybe `get_outstanding_requests_for_output_checker`?

Reply:
Gone with `get_outstanding_requests_for_review`

`..._for_output_checker` implies that they are only for a specific output checker, were as is actually *all* such requests. User is only passed to verify they are indeed an output checker